woocommerce/woocommerce
# Use high-resolution version of wp spinner

### All Submissions:


### Changes proposed in this Pull Request:

- Use high-resolution version of wp spinner

### How to test the changes in this Pull Request:

1. Go to WooCommerce > Orders.
2. Click the Preview icon.
3. You should see the new high-resolution spinner.

### Other information:


### Changelog entry

- Use high-resolution version of wp spinner
